Networking/Networking/Mapper/SitePluginsMapper.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// Mapper: SitePlugins
+///
+struct SitePluginsMapper: Mapper {
+
+    /// Site Identifier associated to the plugins that will be parsed.
+    /// We're injecting this field via `JSONDecoder.userInfo` because the remote endpoints don't return the SiteID in the plugin endpoint.
+    ///
+    let siteID: Int64
+
+    /// (Attempts) to convert a dictionary into [SitePlugin].
+    ///
+    func map(response: Data) throws -> [SitePlugin] {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(DateFormatter.Defaults.dateTimeFormatter)
```

Comment:
nit: I noticed that there are no dates to decode, so the date decoding strategy might be superfluous 👍

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/8e320fa88e27122e36d57a3de6e396bc6ec0ce37